scene/resources/material.cpp

```diff
@@ -2136,6 +2163,8 @@ void SpatialMaterial::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::REAL, "distance_fade_min_distance", PROPERTY_HINT_RANGE, "0,4096,0.01"), "set_distance_fade_min_distance", "get_distance_fade_min_distance");
 	ADD_PROPERTY(PropertyInfo(Variant::REAL, "distance_fade_max_distance", PROPERTY_HINT_RANGE, "0,4096,0.01"), "set_distance_fade_max_distance", "get_distance_fade_max_distance");
 
+	ADD_PROPERTY(PropertyInfo(Variant::INT, "fallback_mode", PROPERTY_HINT_ENUM, "Ubershader,None,No Render"), "set_fallback_mode", "get_fallback_mode");
```

Comment:
Reorder to match the actual enum and improve the hint descriptions:

```suggestion
	ADD_PROPERTY(PropertyInfo(Variant::INT, "fallback_mode", PROPERTY_HINT_ENUM, "None (Synchronous),No Render (Async. Invisible),Ubershader (Async. Visible)"), "set_fallback_mode", "get_fallback_mode");
```

Reply:
I've added your descriptions, but kept my ordering (it seems I had already pushed a change of ordering when you had already seen this).